Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -163,3 +166,27 @@ final class PledgePaymentMethodsViewController: UIViewController {
       |> \.textAlignment .~ .center
   }
 }
+
+extension PledgePaymentMethodsViewController: PledgeAddNewCardViewDelegate {
+  func pledgeAddNewCardViewDidTapAddNewCard(_: PledgeAddNewCardView) {
+    let addNewCardViewController = AddNewCardViewController.instantiate()
+      |> \.delegate .~ self
+    let navigationController = UINavigationController.init(rootViewController: addNewCardViewController)
```

Comment:
We don't need `.init` here.

Reply:
Yeah that's the only way I could get the autocomplete 😓